op-batcher/batcher/config.go

```diff
@@ -107,15 +72,15 @@ func (c CLIConfig) Check() error {
 	if err := c.TxMgrConfig.Check(); err != nil {
 		return err
 	}
-	if err := c.RPCFlag.Check(); err != nil {
+	if err := c.RPC.Check(); err != nil {
 		return err
 	}
 	return nil
 }
 
 // NewConfig parses the Config from the provided flags or environment variables.
-func NewConfig(ctx *cli.Context) CLIConfig {
-	return CLIConfig{
+func NewConfig(ctx *cli.Context) *CLIConfig {
```

Comment:
Why is this a pointer now?

Reply:
To avoid unnecessary copies of the config struct, with all the op-service sub-configs it's gotten pretty large, and it might not be clone-able with just a shallow-copy in the future.